OneCommunityGlobal/HighestGoodNetworkApp
# Yixiao_taskeditsuggestionmodal_unittest

# Description
Create a unit test for the TaskEditSuggestionsModal component to confirm whether it working as intended.


## Main changes explained:
- Create file TaskEditSuggestionsModal.test.js to do the unit test.
- Change some structure of file TaskEditSuggestionsModal.jsx for more friendly test.

## How to test:
1. check into current branch
2. do npm install to run this PR locally
3. run npm test TaskEditSuggestionsModal.test.js

## Screenshots or videos of changes:
